spec/factories/tasks.rb

```diff
@@ -5,5 +5,12 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    association :project
+
+    trait :done_status_change do
```

Comment:
doneがステータス系のカラムであることは予想できるので、trait名も短くて良いかなと思います。
```suggestion
    trait :done do
```

Reply:
こちら 368a944a6a5dbd9a938af59949e83dff39d25c31 で修正しました